### PR TITLE
Update advanced_example notebook 

### PR DESCRIPTION
**Pull request by sararb**
This PR updates the advanced example notebook with the comments from the live demo presentation.

### CODE REVIEW

File: examples/06-Define-your-own-architecture-with-Merlin-Models.ipynb [hunk 1]
```diff
@@ -3,7 +3,7 @@
   {
```
Comment (rnyak) [inline examples/06-Define-your-own-architecture-with-Merlin-Models.ipynb:3]: <p>May be tf.keras fit method instead of Keras?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Comment (rnyak) [inline examples/06-Define-your-own-architecture-with-Merlin-Models.ipynb:3]: <p>May be remove <code>:</code> at the end and add <code>.</code> instead?</p><p> </p><p>Also you can write <code>Let's check out model evaluation scores.</code>  if you want.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Comment (rnyak) [inline examples/06-Define-your-own-architecture-with-Merlin-Models.ipynb:3]: <p>The name of the notebook should be <code>Serving-Ranking-Models-With-Merlin-Systems.ipynb</code> .</p><p> </p><p>I'd rephrase as:</p><p> </p><p> <code>... notebook that deploys an ensemble of&nbsp;<a href="https://github.com/NVIDIA-Merlin/NVTabular" target="_blank">NVTabular</a>&nbsp;workflow and a trained model from Merlin Models to ...</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Comment (sararb) [reply examples/06-Define-your-own-architecture-with-Merlin-Models.ipynb:3]: <p>I like the proposed rephrasing! regarding the usage of the colon instead of a period, I use it for all the instructional text as advised by Mike in the previous revisions.</p>

**Review by rnyak — APPROVED** [review]
